src/main/java/com/iridium/iridiumskyblock/database/LostItems.java

```diff
@@ -0,0 +1,51 @@
+package com.iridium.iridiumskyblock.database;
+
+import com.iridium.iridiumcore.utils.ItemStackUtils;
+import com.iridium.iridiumteams.database.DatabaseObject;
+import com.j256.ormlite.field.DataType;
+import com.j256.ormlite.field.DatabaseField;
+import com.j256.ormlite.table.DatabaseTable;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.bukkit.inventory.ItemStack;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Arrays;
+import java.util.UUID;
+
+@NoArgsConstructor
+@Getter
+@Setter
+@DatabaseTable(tableName = "lost_items")
+public class LostItems extends DatabaseObject {
+    @DatabaseField(columnName = "id", generatedId = true, canBeNull = false, unique = true)
+    private int id;
+
+    @DatabaseField(columnName = "uuid", canBeNull = false)
+    private @NotNull UUID uuid;
+
+    @DatabaseField(columnName = "time")
+    private long time;
+
+    @DatabaseField(columnName = "items", dataType = DataType.SERIALIZABLE)
+    private String[] items;
```

Comment:
Have you tested this? didnt know you could have an array here?

Reply:
Yeah, me and woof tested this. Here are some of the results:

Player message:
![image](https://github.com/Iridium-Development/IridiumSkyblock/assets/41022228/dc47633a-50ad-4cf1-b4da-99345f0c21b7)

Console:
`[23:03:39 INFO]: [IridiumSkyblock] Lost items: Item: LIGHT_BLUE_WOOL, Amount: 20, Name: , Item: YELLOW_WOOL, Amount: 1, Name: , Item: RED_WOOL, Amount: 6, Name: , Item: PURPLE_WOOL, Amount: 5, Name: , Item: LIME_WOOL, Amount: 5, Name:`